_config.yml

```
@@ -1,21 +1,44 @@
 # Site settings
 title: Pyronear
 description: "Détection précoce de feux de forêt" # this means to ignore newlines until "baseurl:"
```

this com is useless now
```suggestion
description: "Détection précoce de feux de forêt"
```

---

Thanks, I'll remove comments in another PR to avoid requesting reviews :ok_hand: